parsl/tests/test_ipp/bash_app.py

```diff
@@ -12,12 +12,15 @@
 # parsl.set_stream_logger()
 
 workers = IPyParallelExecutor()
-dfk = DataFlowKernel(workers)
+dfk = DataFlowKernel(executors=[workers])
 
 
 @App('bash', dfk)
 def echo_to_file(inputs=[], outputs=[], stderr='std.err', stdout='std.out'):
-    pass
+    res = ""
+    for i, o in zip(inputs, outputs):
```

Comment:
This assumesj len(inputs) == len(outputs).
Might be safer to do : 
return "for i in {0}; do echo {1} &> $i ; done".format(" ".join(outputs), " ".join(inputs) )

Reply:
Good point @yadudoc, I've updated it.